.github/ISSUE_TEMPLATE/release.yml

```diff
@@ -99,17 +98,31 @@ body:
       label: Hotfix release checklist
       description: Complete these items in sequence as the `Hotfix` release progresses
       options:
-        - label: Make the hotfix in a branch from `prod`
-        - label: Bump the application version
+        - label: Create a hotfix branch from the latest release tag on `main` (`git checkout -b hotfix YYYY.0M.R`)
+        - label: Commit the fixes to the hotfix branch
         - label: Ensure `prod` secrets are up to date
-        - label: Open a PR from the hotfix branch into `prod`, merge
+        - label: Create an annotated tag on the hotfix branch (`git tag -a YYYY.0M.R`)
+        - label: Push the annotated tag (`git push origin YYYY.0M.R`)
         - label: Smoke Test the fix in prod
         - label: Confirm acceptance of Smoke Tests by adding a comment to this issue
-        - label: Tag the release on `prod`, push the tag to GitHub (see [docs](https://docs.calitp.org/benefits/deployment/release/#5-tag-the-release) for commands)
-        - label: Create a release in GitHub for the tag, generating release notes
         - label: Edit release notes to link back to this Release process issue
         - label: Edit release notes with additional context, images, animations, etc. as-needed
-        - label: Create a branch off `main`
-        - label: Open a PR from `prod` into that branch, merge
-        - label: (optional) Adapt hotfix to current state of `main` in that same branch
-        - label: Merge the PR into `main`
+        - label: (optional) Open a PR from the hotfix branch to the current state of `main`
+        - label: (optional) Merge the PR into `main`
+  - type: checkboxes
+    id: rollback-checklist
+    attributes:
+      label: Rollback release checklist
+      description: Complete these items in sequence as the `Rollback` release progresses
+      options:
+        - label: Create a rollback branch from the last known good commit on `main` (`git checkout -b rollback <commit-hash>`)
+        - label: Ensure `prod` secrets are up to date
+        - label: Create an annotated tag on the HEAD of the rollback branch (`git tag -a YYYY.0M.R`)
+        - label: Push the annotated tag (`git push origin YYYY.0M.R`)
+        - label: Smoke Test the fix in prod
+        - label: Confirm acceptance of Smoke Tests by adding a comment to this issue
+        - label: Edit release notes to link back to this Release process issue
+        - label: Edit release notes with additional context, images, animations, etc. as-needed
+        - label: (optional) Fix whatever issue is wrong using the rollback branch
```

Comment:
I'm a little confused by this part and wrote out my question on the [original ticket](https://github.com/cal-itp/benefits/issues/1249#issuecomment-2270038816)

Reply:
@angela-tran and @thekaveman thanks for clearing this up in the [original ticket](https://github.com/cal-itp/benefits/issues/1249#issuecomment-2270038816). I am updating the checklist and realized that we probably don't even need to initially create a `rollback` branch, we just need to tag (with the rollback version) the commit associated with the last known good release tag (since it's ok for one commit to have more than one tag). I guess that  avoiding creating a `rollback` branch at this point will keep our repo cleaner with less branches, especially because this branch won't get deleted later (it'll behave similar to `test` and `prod` like we talked about in [this other comment](https://github.com/cal-itp/benefits/pull/2270#issuecomment-2265916700)). So if it makes sense, I'd propose the following process (wanted to get your thoughts here first before asking for a complete re-review 🙂):

- Create an annotated tag (with the rollback release version) on the commit associated with the last known good release tag on `main` (`git tag -a YYYY.0M.R <commit-hash>`)
- Ensure `prod` secrets are up to date
- Push the annotated tag (`git push origin YYYY.0M.R`)
- Smoke Test the rollback in prod
- Confirm acceptance of Smoke Tests by adding a comment to this issue
- Edit release notes to link back to this Release process issue
- Edit release notes with additional context, images, animations, etc. as-needed
- (optional) Create a `rollback-fix` branch from the commit on `main` that prompted the rollback (`git checkout -b rollback-fix <commit-hash>`)
- (optional) Commit the fixes to the `rollback-fix` branch
- (optional) Open a PR from the `rollback-fix` branch to `main`
- (optional) Merge the PR into `main`

Oh and the last 4 steps to implement a fix I feel would be optional since they don't need to happen during the release session, they will typically take longer (contrary to a hotfix) so they will probably take place after the release session.

Reply:
> especially because this branch won't get deleted later (it'll behave similar to test and prod like we talked about in https://github.com/cal-itp/benefits/pull/2270#issuecomment-2265916700).

I don't think this is the case for a `rollback` branch (or really any other branch going forward): we tag commits to mark the point that a release occurred. In the case of a `rollback`, we are tagging a commit that is _already on `main`_ (from a previous release) so that new tag is also on `main`. If we then make a branch to fill the cause of the `rollback`, merge that into `main` and tag for a new release -- that's just our standard PR process (and the branch is deleted at the end).

For a `hotfix`, yes we create the branch and new commits and tag the `HEAD` once we're done... but eventually that `hotfix` branch is merged back into `main`, those commits and the tag get into `main`, and it again is similar to our normal PR process. 

So I would not expect us to have any more long-lived branches like `test` and `prod`. Our goal should be to get those changes/fixes into `main` eventually. Having to deal with `N` long-lived branches (for `N` rollbacks/`N` hotfixes) would be a nightmare!

> Oh and the last 4 steps to implement a fix I feel would be optional

I agree, I don't even think it makes sense to list them on the checklist. A `rollback` is meant to quickly restore the app to a known good state, it is an entirely separate effort to address the root cause of why the rollback was needed.